gpiozero/internal_devices.py

```diff
@@ -504,8 +507,35 @@ class TimeOfDay(PolledInternalDevice):
 
         pause()
 
-    Note that *start_time* may be greater than *end_time*, indicating a time
-    period which crosses midnight.
+    By default start and end times are timezone-aware UTC times. If you wish to
+    specify the time-zone for the start and/or end time you can do so when 
+    contructing the time, for example to switch on when it is office hours in 
+    both London and Los Angeles::
+
+        from gpiozero import TimeOfDay,
+        from datetime import time,
+        from zoneinfo import ZoneInfo
+
+        tz_LA = ZoneInfo('America/Los_Angeles')
+        tz_London = ZoneInfo('Europe/London')
+
+        officehours = TimeOfDay(time(8,30,tzinfo=tz_LA), time(18,00,tzinfo=tz_London))
```

Comment:
I think this example of using different timezones for the start-time and end-time is a bit confusing?

Reply:
The general concept or the way the code is written? I felt the example was quite powerful, but maybe the initial paragraph above needs some work?
Would it be more readable as:

```python
tz_LA = ZoneInfo('America/Los_Angeles')
tz_London = ZoneInfo('Europe/London')

LA_opens = time(8,30,tzinfo=tz_LA)
London_closes = time(18,00,tzinfo=tz_London)

officehours = TimeOfDay(LA_opens, London_closes)
```

Reply:
I think the general concept of using different timezones for the start-time and end-time is a bit confusing.

I think your example assumes that the reader knows that the LA timezone is "negative" compared to the London timezone? And does it assume that LA doesn't close before London opens?